tests/spec/fs.writeFile-readFile.spec.js

```diff
@@ -22,6 +22,19 @@ describe('fs.writeFile, fs.readFile', function() {
     });
   });
 
+  
+  it('should error when path is wrong to writeFile',function(done){
```

Comment:
Just to nitpick here, you can use ()=>{....} to declare a function instead of function().